### PR TITLE
Update Java logging with resource names

### PR DESCRIPTION
**Pull request by michaelbausor**
See this PR for gapic yaml update: https://github.com/googleapis/googleapis/pull/208

### CODE REVIEW

File: generated/java/google-cloud-logging-v2/src/main/java/com/google/cloud/logging/spi/v2/ConfigServiceV2Client.java [hunk 1]
```diff
@@ -211,8 +213,8 @@ public final ConfigServiceV2Settings getSettings() {
    *
    * <pre><code>
    * try (ConfigServiceV2Client configServiceV2Client = ConfigServiceV2Client.create()) {
-   *   String formattedParent = ConfigServiceV2Client.formatParentName("[PROJECT]");
-   *   for (LogSink element : configServiceV2Client.listSinks(formattedParent).iterateAllElements()) {
+   *   ParentNameOneof parent = ParentNameOneof.from(ParentName.create("[PROJECT]"));
```
Comment (michaelbausor) [inline generated/java/google-cloud-logging-v2/src/main/java/com/google/cloud/logging/spi/v2/ConfigServiceV2Client.java]: Yes, I think we should.

**Comment by michaelbausor** [comment]
Updated, PTAL

**Comment by michaelbausor** [comment]
Updated with latest changes, PTAL. Note also that these resource name changes will require a bump and republish of the grpc packages

**Comment by garrettjonesgoogle** [comment]
No longer WIP :-) LGTM.

**Pull request merged**